app/controllers/application_controller.rb

```diff
@@ -74,15 +74,15 @@ def instance_link(instance)
     send("api_#{version}_#{endpoint}_url", instance.id)
   end
 
-  def raise_event_if(ignore_raise_event, event, payload)
-    return if ignore_raise_event
+  def raise_event_if(raise_event, event, payload)
+    return unless raise_event
```

Comment:
yes `ignore_raise_event` was not great but what about to name this more like `raise_event_allowed` or  `raise_event_expression` or `raise_event_condition`
or something like that express that it is condition

Reply:
Changed to `raise_event_allowed` :)